src/main/kotlin/com/jay/racingcar/domain/Name.kt

```diff
@@ -0,0 +1,11 @@
+package com.jay.racingcar.domain
+
+class Name(val value: String) {
+    init {
+        require(value.length <= MAX_NAME_LENGTH) { "이름은 5글자를 초과할 수 없습니다." }
+    }
+
+    companion object {
+        const val MAX_NAME_LENGTH = 5
```

Comment:
```suggestion
        private const val MAX_NAME_LENGTH = 5
```
클래스 내부에서만 접근하는 것이라면, private를 붙여주는 습관을 들여보면 어떨까요?